<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashScreen</class>
 <widget class="QMainWindow" name="SplashScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Anti-AFK - Waiting for ROBLOX to open</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="accessibleDescription">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.499955, fy:0.5, stop:0 rgba(88, 93, 141, 255), stop:1 rgba(56, 58, 89, 255));
	color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="title_frame">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>381</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>381</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Roboto Black</family>
          <pointsize>40</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(134, 160, 255);</string>
        </property>
        <property name="text">
         <string>ANTI-AFK</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>381</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Roboto Medium</family>
          <pointsize>13</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(134, 160, 255);</string>
        </property>
        <property name="text">
         <string>FOR &lt;strong&gt;&lt;i&gt;ROBLOX&lt;/i&gt;&lt;/strong&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="loading_frame">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>180</y>
         <width>381</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="loading_text">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>381</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Waiting for &lt;strong&gt;ROBLOX&lt;/strong&gt; to open</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="credit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>260</y>
         <width>381</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Roboto Thin</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(96, 100, 153);
background-color: transparent;</string>
       </property>
       <property name="text">
        <string>&lt;strong&gt;Made by:&lt;/strong&gt; Carl Matchu</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>-1</number>
       </property>
       <property name="indent">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="loading">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>210</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>381</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QFrame" name="main_frame">
       <property name="geometry">
        <rect>
         <x>-320</x>
         <y>110</y>
         <width>321</width>
         <height>111</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QCheckBox" name="anti_afk">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>40</y>
          <width>30</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Roboto Medium</family>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator {
	width: 30px;
	height: 30px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="shortcut">
         <string>Space</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>281</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Roboto Medium</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Anti-AFK</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="status">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>321</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>STATUS: &lt;strong&gt;not active&lt;/strong&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <zorder>label_3</zorder>
       <zorder>anti_afk</zorder>
       <zorder>status</zorder>
      </widget>
      <widget class="QPushButton" name="exit_app">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>15</y>
         <width>25</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Impact</family>
         <pointsize>25</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Exit</string>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>×</string>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="always_on_top">
       <property name="geometry">
        <rect>
         <x>305</x>
         <y>15</y>
         <width>25</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Impact</family>
         <pointsize>25</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Always on top</string>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
image: url(:/resources/resources/always_on_top.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
